Resources/public/js/views/ez-trashview.js

```diff
@@ -88,6 +89,27 @@ YUI.add('ez-trashview', function (Y) {
             });
         },
 
+        /**
+         * Restores the selected trash items
+         *
+         * @private
+         * @method _restoreTrashItems
+         */
+        _restoreTrashItems: function (e) {
+            var selectedTrashItems = Y.all('.ez-trashitem-box:checked'),
```

Comment:
don't use `Y.all`, a view is only responsible what happens in its own container so here you should write `this.get('container')..all('.ez-trashitem-box:checked');